src/main/java/com/openlattice/chronicle/configuration/TwilioConfiguration.kt

```diff
@@ -0,0 +1,32 @@
+package com.openlattice.chronicle.configuration
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.kryptnostic.rhizome.configuration.Configuration
+import com.kryptnostic.rhizome.configuration.ConfigurationKey
+import com.kryptnostic.rhizome.configuration.SimpleConfigurationKey
+import com.kryptnostic.rhizome.configuration.annotation.ReloadableConfiguration
+
+private const val CONFIG_FILE_NAME = "twilio.yaml"
+@ReloadableConfiguration(uri = CONFIG_FILE_NAME)
+
+class TwilioConfiguration @JsonCreator constructor(
+    @JsonProperty val sid: String,
+    @JsonProperty val token: String,
+    @JsonProperty val fromPhone: String
+) : Configuration {
+
+    @JsonIgnore
+    override fun getKey(): ConfigurationKey {
+        return Companion.key
+    }
+
+    companion object {
+        protected var key: ConfigurationKey = SimpleConfigurationKey(CONFIG_FILE_NAME)
+        @JsonIgnore
+        fun key(): ConfigurationKey {
+            return key
+        }
+    }
```

Comment:
What is `fun key()` for? I don't think this is necessary. Additionally, you can simplify the companion object block.

```
companion object {
  @JvmStatic
  val key = SimpleConfigurationKey(CONFIG_FILE_NAME)
}
```